src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs

```diff
@@ -215,6 +217,18 @@ public static void Verify_AccessFieldClass()
         extern static ref string GetPrivateField(UserDataClass d);
     }
 
+    [Fact]
+    public static void Verify_AccessFieldOffsetClass()
+    {
+        Console.WriteLine($"Running {nameof(Verify_AccessFieldOffsetClass)}");
+
+        // Offset = method table pointer + first string pointer
+        Assert.Equal(IntPtr.Size + IntPtr.Size, GetPrivateFieldOffset());
```

Comment:
This should just be `IntPtr.Size` I think, since you cannot legitimately obtain a ref before the first field iirc.